torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -405,9 +413,6 @@ def __init__(
             _check_orig_params_flattened(self, self._ignored_params)
             _register_flat_param(self, self)
 
-        # Delete to avoid keeping references after the constructor
-        delattr(self, "_ignored_params")
```

Comment:
remove this, as we need to get the '_ignored_params' of nested FSDP instance. This should not result in memory leak...? as they are part of and referenced by module parameters() anyway during lifetime of training

Reply:
Yes, this should be fine. `use_orig_params=True` path assumes that the user does not swap parameter variables after initialization, which is the same assumption here to not have a memory leak.